example-projects/rdkafka-example/src/main.rs

```diff
@@ -0,0 +1,160 @@
+use clap::{App, Arg};
+use futures::StreamExt;
+use serde_json::json;
+
+use cloudevents::{EventBuilder, EventBuilderV10};
+use cloudevents_sdk_rdkafka::{FutureRecordExt, MessageExt, MessageRecord};
+
+use rdkafka::config::{ClientConfig, RDKafkaLogLevel};
+use rdkafka::consumer::stream_consumer::StreamConsumer;
+use rdkafka::consumer::{CommitMode, Consumer, DefaultConsumerContext};
+use rdkafka::producer::{FutureProducer, FutureRecord};
+
+// run a Kafka lense or a Kafka Docker container to try out this example
```

Comment:
Why did you removed the docker command? it was a good one, just the name is not kafka lense :smile: 

Maybe rephrase with:

> You need a running Kafka cluster to try out this example. With docker: [insert command her